### PR TITLE
feat: Nft details transfers tab

### LINKED ISSUE #147: NFT details transfers tab
A new tab called "Transfers" should be visible in the NFT details page with the following paginated table:
| Hash | Time | Token ID | Recipient | Sender |
|--------|--------|--------|--------|--------|
| th_k67Jg5p...n7 | 1 day ago 651434 | 8 | ak_QVSUoGr...YW | ak_2tw26Rw...z8 |

Example call: https://testnet.aeternity.io/mdw/v2/aex141/transfers/ct_2tw26RwgNADrpuCnrQWKPBH87bPxuRbLR1KLccS9ZJTUMMj4z8

### PR DESCRIPTION
**Pull request by janmichek**
## Description
fixes #147

## Demo
`ct_2tw26RwgNADrpuCnrQWKPBH87bPxuRbLR1KLccS9ZJTUMMj4z8`
image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add transfer nft tab

### CODE REVIEW

**Comment by janmichek** [comment]
Do we have any living example of an nft with transfer records? @michele-franchi

**Comment by michele-franchi** [comment]
I'll review it once NFT list and detail PRs are merged

File: src/components/NftsPanel.vue [hunk 1]
```diff
@@ -5,7 +5,6 @@
       :entities="nfts"
       :total-count="nftsCount"
       :limit="limit"
-      pagination-style="history"
```
Comment (janmichek) [inline src/components/NftsPanel.vue:8]: fix from previous work, there should not be history labels as the list is not sorted by the date yet

**Comment by janmichek** [comment]
Let's continue in here https://github.com/aeternity/aescan/pull/494

**Pull request closed without merging**